#! /bin/sh /usr/share/dpatch/dpatch-run
## 92-resmgr-template.dpatch by Peter Eisentraut <petere@debian.org>
##
## DP: Adjustments in resmgr template

@DPATCH@
diff -urNad kolabd-2.1.0-20070510.dfsg~/templates/resmgr.conf.template.in kolabd-2.1.0-20070510.dfsg/templates/resmgr.conf.template.in
--- kolabd-2.1.0-20070510.dfsg~/templates/resmgr.conf.template.in	2007-05-08 11:39:04.000000000 +0200
+++ kolabd-2.1.0-20070510.dfsg/templates/resmgr.conf.template.in	2007-08-08 15:21:23.000000000 +0200
@@ -6,7 +6,7 @@
 <?php
 
 // What is the root directory of our Horde instance?
-@define('HORDE_BASE', '@webserver_document_root@@webserver_web_prefix@/fbview');
+@define('HORDE_BASE', '/usr/share/horde3');
 
 // What is the address of the Cyrus server where the calendar data is stored?
 $params['server'] = 'localhost';
